src/App.css

```diff
@@ -1,33 +1,75 @@
-.App {
-  text-align: center;
+button{
+  background-color:#1db954;
+  width: 200px;
+  padding: 18px 0 16px 0px;
+  border-radius: 500px;
+  color:white;
+  font-family: Roboto, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif , Arial, Helvetica, sans-serif;
+  letter-spacing: 3px;
+  font-size: 14px;
+  font-weight: 700;
+  border: 0;
+  box-shadow: 0;
+  margin: auto;
+
+cursor: pointer;
+}
+
+button:hover{
+  background-color: #1ed760
 }
 
-.App-logo {
-  animation: App-logo-spin infinite 20s linear;
-  height: 40vmin;
-  pointer-events: none;
+.App{
+  width: 100%;
+  text-align: center;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+.Pla_Pan{
```

Comment:
No reason to abbreviate classnames. Go for meaningful names and think of who will have to read your code